spec/system/admin/reports_spec.rb

```diff
@@ -476,12 +481,14 @@
 
       it "can customise a number of fields" do
         fill_in 'initial_invoice_number', with: '5'
-        fill_in 'invoice_date', with: '2015-02-12'
-        fill_in 'due_date', with: '2015-03-12'
+        
+        pick_datetime '#invoice_date', Date.new(2021, 2, 12)
+        pick_datetime '#due_date', Date.new(2021, 3, 12)
```

Comment:
Hm, if you use `pick_datetime` with a Date instead of `pick_date` then we don't actually need both methods. :stuck_out_tongue: